apps/communications/dialer/test/marionette/lib/dialer.js

```diff
@@ -41,11 +45,21 @@ Dialer.Selectors = {
   callLogNoResultsContainer: '#no-result-container',
   callLogItem: '.log-item',
   callLogEditForm: '#edit-mode',
+  noEntriesMessage: '#no-result-msg1',
```

Comment:
From here [to there](https://github.com/mozilla-b2g/gaia/pull/33913/files#diff-ef4efacac7cfe3c475dc13f5c99492acR62), none of the selectors you defined is used. Even though they might be useful in the foreseeable , I'd get them out of the PR because of 2 reasons: consistency and [YAGNI](https://en.wikipedia.org/wiki/YAGNI)